loyalty_incompatibility/__manifest__.py

```diff
@@ -4,13 +4,13 @@
 {
     "name": "Loyalty incompatibilities",
     "summary": "Allows to set incompatibility rules between promotions",
-    "version": "16.0.1.0.0",
+    "version": "18.0.1.0.0",
     "development_status": "Production/Stable",
     "category": "Sale",
     "website": "https://github.com/OCA/sale-promotion",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "maintainers": ["chienandalu"],
     "license": "AGPL-3",
-    "depends": ["loyalty"],
+    "depends": ["loyalty", "sale"],
```

Comment:
The `sale` dependency should not be added, the `sale_loyalty_incompatibility` module already has that target.